public/app/screens/main/sections/devices/device-finder/style.scss

```diff
@@ -0,0 +1,41 @@
+@import "theme";
+
+.cc-device-finder {
+  &__spinner {
+    background-image: url("./img/spinner.svg");
+    background-repeat: no-repeat;
+    background-size: 100%;
+    width: 50px;
+    height: 50px;
+
+    -webkit-animation-name: rotate;
+    -webkit-animation-duration:2s;
```

Comment:
We have auto prefixer loader so we don't need to specify browser specific prefixes in scss.